deployment/terraform/create.go

```diff
@@ -678,6 +685,19 @@ func (t *Terraform) updateAppConfig(siteURL string, sshc *ssh.Client, jobServerE
 		}
 	}
 
+	if t.output.HasKeycloak() {
+		keycloakScheme := "http"
+		if !t.config.ExternalAuthProviderSettings.DevelopmentMode {
+			keycloakScheme = "https"
+		}
+
+		cfg.OpenIdSettings.Enable = model.NewBool(true)
+		cfg.OpenIdSettings.ButtonText = model.NewString("Keycloak Login")
+		cfg.OpenIdSettings.DiscoveryEndpoint = model.NewString(keycloakScheme + "://" + t.output.KeycloakServer.PublicDNS + ":8080/realms/" + t.config.ExternalAuthProviderSettings.KeycloakRealmName + "/.well-known/openid-configuration")
```

Comment:
I see this `8080` being duplicated in a lot of places. Not for now, but later we'd want to move this to a config and use that value instead.